src/modules/recipe/externalComponentCdn/externalComponentCdn.js

```diff
@@ -0,0 +1,19 @@
+import { LightningElement } from 'lwc';
+
+export default class ExternalComponentCDN extends LightningElement {
+    rating = 3;
+    rendered = false;
+
+    renderedCallback() {
+        if (this.rendered) {
+            return;
+        }
+        this.rendered = true;
+
+        this.template
+            .querySelector('sl-rating')
+            .addEventListener('sl-change', (event) => {
```

Comment:
Is there a reason why you add the event listener programmatically via JS instead of having it in the HTML?
If this is a requirement, I recommend mentioning it in a comment. Otherwise, move it to HTML so that we're consistent with other recipes.

Reply:
LWC's declarative event bindings only support lowercase events and don't support hyphens. Added a comment in the code for this.